UI/FileSavePickeriOS/FileSavePickeriOS/FileSavePickeriOS.Shared/Strings/en/Resources.resw

```diff
@@ -0,0 +1,168 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Quieres la traduccion en Español?  :)

Reply:
Definitely :-) ! If you can provide me the translated strings, I will add them to the PR :-) (or you can commit to this PR directly if you have access :-) )